include/deal.II/lac/utilities.h

```diff
@@ -226,7 +226,8 @@ namespace Utilities
         {
```

Comment:
please fix the commit message of this commit: it's "deprecated", not "depricated"